analysis/tools/ccsh/src/main/kotlin/de/maibornwolff/codecharta/tools/ccsh/Ccsh.kt

```diff
@@ -75,15 +87,80 @@ class Ccsh : Callable<Void?> {
         fun executeCommandLine(args: Array<String>): Int {
             val commandLine = CommandLine(Ccsh())
             commandLine.executionStrategy = CommandLine.RunAll()
-            if (args.isEmpty() || isParserUnknown(args, commandLine)) {
-                return executeInteractiveParser(commandLine)
+            return if (args.isEmpty()) {
+                val configuredParsers = offerInteractiveParserSuggestions(commandLine)
+                if (configuredParsers.isEmpty()) {
+                    return 0
+                }
+
+                return executeConfiguredParsers(commandLine, configuredParsers)
+            } else if (isParserUnknown(args, commandLine) || args.contains("--interactive") || args.contains("-i")) {
+                executeInteractiveParser(commandLine)
             } else {
-                return commandLine.execute(*sanitizeArgs(args))
```

Comment:
There is no value returned anymore in this case.

Reply:
I moved the return in front of the if - should this not just return the old value or did I understand this wrong?